.github/workflows/ci.yaml

```diff
@@ -231,7 +231,7 @@ jobs:
         uses: actions/cache@d4323d4df104b026a6aa633fdb11d772146be0bf # v4
         with:
           path: ${{env.transcript_test_results}}
-          key: transcripts-results-${{ matrix.os }}-${{ hashFiles('**/ci.yaml', '**/stack.yaml', '**/package.yaml', '**/*.hs')}}-${{ hashFiles('**/unison-src/**/*.md', '**/unison-src/**/*.u') }}
+          key: transcripts-results-${{ matrix.os }}-${{ hashFiles('**/ci.yaml', '**/stack.yaml', '**/package.yaml', '**/*.hs')}}-${{ hashFiles('**/unison-src/**/*.md', '**/unison-src/**/*.u', '**/docs/*.md', '**/docs/*.markdown') }}
```

Comment:
This is a good idea, but maybe we should also just rename all the `*.markdown` -> `*.md` for consistency so this doesn't happen again in other scripts?